src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ScheduleRuleService.kt

```diff
@@ -12,13 +12,10 @@ import java.time.temporal.TemporalAdjusters
 @Service
 class ScheduleRuleService {
 
-  fun mapRules(dto: CreateActivityRequest, courseActivity: CourseActivity) =
-    dto.scheduleRules.map {
+  fun mapRules(dto: CreateActivityRequest, courseActivity: CourseActivity): List<CourseScheduleRule> {
+    val monthStart = LocalDate.now().with(TemporalAdjusters.firstDayOfMonth())
+    return dto.scheduleRules.map {
       validateRequest(it)
-
-      // Prod data usually uses 1st day of the scheduleStartDate month for the 'date' part
-      val monthStart = courseActivity.scheduleStartDate!!.with(TemporalAdjusters.firstDayOfMonth())
```

Comment:
Further testing revealed this is untrue - Nomis uses the 1st day of the current month